Sources/CognitoIdentityDemo/CognitoIdentityDemo.swift

```diff
@@ -21,15 +22,21 @@ class CognitoIdentityDemo {
         cognitoIdentityClient = try CognitoIdentityClient()
     }
     
-    /// Returns the ID of the identity pool with the specified name.
+    /// Returns the ID of the identity pool with the specified name by calling the completion handler with
```

Comment:
Please remove the call to:
```
        SDKLoggingSystem.add(logHandlerFactory: CognitoIdentityClientLogHandlerFactory(logLevel: .info))
```
from the `init` function.  This sort of call should be left up to the developer who is instantiating `CognitoIdentityDemo`